editor/editor_inspector.cpp

```diff
@@ -3488,6 +3503,10 @@ void EditorInspector::_property_selected(const String &p_path, int p_focusable)
 	emit_signal(SNAME("property_selected"), p_path);
 }
 
+void EditorInspector::_property_open_documentation(const String &p_doc_path) {
+	EditorNode::get_singleton()->open_documentation(p_doc_path);
```

Comment:
Can be replaced by `ScriptEditor::get_singleton()->goto_help()`, see SceneTreeDock code.
This way you can avoid modifying EditorNode.

Reply:
I changed it to use `ScriptEditor::get_singleton()->goto_help()` directly in EditorProperty. All of the changes in EditorNode and ScriptEditor are gone now.